packages/grafana-data/src/transformations/transformers/seriesToColumns.ts

```diff
@@ -16,55 +16,64 @@ export const seriesToColumnsTransformer: DataTransformerInfo<SeriesToColumnsOpti
     byField: 'Time',
   },
   transformer: options => (data: DataFrame[]) => {
-    const optionsArray = options.byField ? [options.byField] : [];
-    // not sure if I should use filterFieldsByNameTransformer to get the key field
-    const keyDataFrames = filterFieldsByNameTransformer.transformer({
-      include: optionsArray,
-    })(data);
-    if (!keyDataFrames.length) {
-      // for now we only parse data frames with 2 fields
-      return data;
+    const keyFieldMatch = options.byField || 'Time';
```

Comment:
Maybe this is better?
```suggestion
    const keyFieldMatch = options.byField ?? defaultOptions.byField;
```

Reply:
ah, good catch!